tools/authoring-decisions.adoc

```diff
@@ -3,10 +3,437 @@ include::../partials/attributes.adoc[]
+
+In the example above, the user has created a new DRD and has it renamed to DRD2, which we can see on 1. Clicking on 2 will open the DRG nodes (3). In this example, the “My Decision” and “Person” nodes are present on the DRG but not in the current DRD. By clicking on it and dragging it into the diagram the node will be added to the current DRD. This behavior is illustrated below.
+
+.Dragging a new node from the DRG nodes list
+image::tools/authoringDecisions/new-editor-diagram-drg-nodes-dragging.gif[width=500px]
+
+It's possible to see that the “My Decision” became grayish on the DRG nodes modal, as it's now present on the current DRD. Also, it has a “three dot” mark on the bottom of it, which tells us, that this node has dependencies that are not shown on the current DRD. By dragging the “Person” node we can see the three dots disappearing, and an information requirement is automatically placed.
+
+.DRD alteranative representation for the Input Data node
+image::tools/authoringDecisions/new-editor-diagram-drd-overview-alternative-input.png[width=500px]
+
+Also, it's possible to change the classic shape to an alternative one, making it more visually cohesive with the BPMN model. This change affects the entire DRD, so it's not possible to have just one node with an alternative shape.
+
+==== 4.Properties panel
+
+The properties panel lets you manage the properties of a Decision element. Enabling the possibility to add detailed descriptions, and documentation links and tweak the font and shape styles.
+
+==== 5.Overlay properties
+
+On the Overlay properties, it's possible to tweak user preferences, which helps with visualization and organization.
+
+. Snapping -> Makes the elements snap into an invisible grid. It's possible to tweak the grid layout by changing the "Horizontal" and "Vertical" values.
+
+.Enanle/disable snapping
+image::tools/authoringDecisions/new-editor-overlay-snapping.gif[]
+
+. Highlight selected node(s) hierarchy -> This option is very useful for complex models where the user needs to verify which nodes are somehow impacting the selected node.
+
+.Enanle/disable node hierarchy
+image::tools/authoringDecisions/new-editor-overlay-hierarchy.gif[]
+
+. Show data type toolbar on nodes -> Control if the data type toolbar is visible or not on the Input Data, Decision, BKM, and Decision Service nodes.
+
+.Enable/disable data type toolbar on nodes
+image::tools/authoringDecisions/new-editor-overlay-show-data-type.gif[]
+
+. Enable styles -> It's possible to turn on/off the node styles. It's important to notice that this option will not change the file, only the visual representation.
+
+.Enable/disable data type toolbar on nodes
+image::tools/authoringDecisions/new-editor-overlay-style.gif[]
+
+==== 6.Autolayout (beta)
+
+The Autolayout is still a beta feature. It will automatically organize your model placing the nodes in a way they're close to the ones they have a relationship.
```

Comment:
It will automatically organize your model placing the nodes in a way they're close to the ones they have a relationship. -> It will automatically organize your model by arranging related nodes closer to each other.